src/ca65/instr.c

```diff
@@ -373,6 +381,153 @@ static const struct {
+      { "AND",  0x080266C, 0x20, 0, C39PutArith }, // C39: (ind,x) not allowed
+      { "ASL",  0x000006e, 0x02, 1, PutAll },
+      { "ASR",  0x000006e, 0x3A, 1, PutAll }, // C39
+      { "BAR",  0x0000008, 0xE2, 0, C39PutBARBAS }, // C39
+      { "BAS",  0x0000008, 0xF2, 0, C39PutBARBAS }, // C39
+      { "BBR",  0x0000004, 0x0F, 0, C39PutBBRBBS }, // use #immed for bit
+      { "BBR0", 0x0000000, 0x0F, 0, PutBitBranch }, // amode wrong!?
+      { "BBR1", 0x0000000, 0x1F, 0, PutBitBranch },
+      { "BBR2", 0x0000000, 0x2F, 0, PutBitBranch },
+      { "BBR3", 0x0000000, 0x3F, 0, PutBitBranch },
+      { "BBR4", 0x0000000, 0x4F, 0, PutBitBranch },
+      { "BBR5", 0x0000000, 0x5F, 0, PutBitBranch },
+      { "BBR6", 0x0000000, 0x6F, 0, PutBitBranch },
+      { "BBR7", 0x0000000, 0x7F, 0, PutBitBranch },
+      { "BBS",  0x0000004, 0x8F, 0, C39PutBBRBBS }, // use #immed for bit
+      { "BBS0", 0x0000000, 0x8F, 0, PutBitBranch }, // amode wrong?
+      { "BBS1", 0x0000000, 0x9F, 0, PutBitBranch },
+      { "BBS2", 0x0000000, 0xAF, 0, PutBitBranch },
+      { "BBS3", 0x0000000, 0xBF, 0, PutBitBranch },
+      { "BBS4", 0x0000000, 0xCF, 0, PutBitBranch },
+      { "BBS5", 0x0000000, 0xDF, 0, PutBitBranch },
+      { "BBS6", 0x0000000, 0xEF, 0, PutBitBranch },
+      { "BBS7", 0x0000000, 0xFF, 0, PutBitBranch },
+      { "BCC",  0x0020000, 0x90, 0, PutPCRel8 },
+      { "BCS",  0x0020000, 0xb0, 0, PutPCRel8 },
+      { "BEQ",  0x0020000, 0xf0, 0, PutPCRel8 },
+      { "BIT",  0x0A0006C, 0x00, 2, PutAll }, 
+      { "BMI",  0x0020000, 0x30, 0, PutPCRel8 },
+      { "BNE",  0x0020000, 0xd0, 0, PutPCRel8 },
+      { "BPL",  0x0020000, 0x10, 0, PutPCRel8 },
+      { "BRA",  0x0020000, 0x80, 0, PutPCRel8 },
+      { "BRK",  0x0000001, 0x00, 0, PutAll },
+      { "BVC",  0x0020000, 0x50, 0, PutPCRel8 },
+      { "BVS",  0x0020000, 0x70, 0, PutPCRel8 },
+      { "CLC",  0x0000001, 0x18, 0, PutAll },
+      { "CLD",  0x0000001, 0xd8, 0, PutAll },
+      { "CLI",  0x0000001, 0x58, 0, PutAll },
+      { "CLV",  0x0000001, 0xb8, 0, PutAll },
+      { "CLW",  0x0000001, 0x52, 0, PutAll }, // C39
+      { "CMP",  0x080266C, 0xc0, 0, C39PutArith }, // C39: (ind,x) not allowed
+      { "CPX",  0x080000C, 0xe0, 1, PutAll },
+      { "CPY",  0x080000C, 0xc0, 1, PutAll },
+      { "DEC",  0x000006F, 0x00, 3, PutAll },
+      { "DEX",  0x0000001, 0xca, 0, PutAll },
+      { "DEY",  0x0000001, 0x88, 0, PutAll },
+      { "EOR",  0x080266C, 0x40, 0, C39PutArith }, // C39: (ind,x) not allowed
+      { "EXC",  0x0000020, 0xd3, 0, C39PutExchange }, // C39 
+      { "INC",  0x000006f, 0x00, 4, PutAll },
+      { "INI",  0x0000001, 0xbb, 0, PutAll }, // C39
+      { "INX",  0x0000001, 0xe8, 0, PutAll },
+      { "INY",  0x0000001, 0xc8, 0, PutAll },
+      { "JMP",  0x0010808, 0x4c, 6, PutAll }, // C39: (abs,x) allowed..
+      { "JPI",  0x0000001, 0x0c, 0, PutAll }, // C39
```

Comment:
Greg, thanks for your hint, my IDA plugin shows it this way also.
It seems I have not checked this insn correctly..

ROM:89BD                 JPI     (0x89CA)
ROM:89C0                 JPI     (0x89C8)
ROM:89C3                 JPI     (0x89C6)